### PR TITLE
simple error_log Log Handler

### PR DESCRIPTION
**Pull request by rwillert**
**Description**
A simple Log Handler for error_log.
We use this Handler to log to stdout for Docker Container.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Log/Handlers/ErrorlogHandler.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	 *
+	 * @param array $config
+	 */
+	public function __construct(array $config = [])
+	{
+		parent::__construct($config);
+	}
+
+	//--------------------------------------------------------------------
+
```
Comment (paulbalandan) [inline system/Log/Handlers/ErrorlogHandler.php:31]:
```suggestion
```

No need to call parent constructor if no special initializations will be made by extending class.

File: system/Log/Handlers/ErrorlogHandler.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	 * @return boolean
+	 */
+	public function handle($level, $message): bool
+	{
+		$msg = strtoupper($level) . ' --> ' . $message . "\n";
+
+		return error_log($msg);
+	}
+
+	//--------------------------------------------------------------------
```
Comment (paulbalandan) [inline system/Log/Handlers/ErrorlogHandler.php:50]:
```suggestion
```
Remove

File: tests/system/Log/Handlers/ErrorlogHandlerTest.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+use CodeIgniter\Test\Mock\MockLogger as LoggerConfig;
+
+class ErrorlogHandlerTest extends \CodeIgniter\Test\CIUnitTestCase
+{
+
+	protected function setUp(): void
+	{
+		parent::setUp();
+	}
+
```
Comment (paulbalandan) [inline tests/system/Log/Handlers/ErrorlogHandlerTest.php:13]:
```suggestion
```
No need.

File: tests/system/Log/Handlers/ErrorlogHandlerTest.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?php
+namespace CodeIgniter\Log\Handlers;
+
+use CodeIgniter\Test\Mock\MockLogger as LoggerConfig;
+
+class ErrorlogHandlerTest extends \CodeIgniter\Test\CIUnitTestCase
```
Comment (paulbalandan) [inline tests/system/Log/Handlers/ErrorlogHandlerTest.php:6]:
```suggestion
use CodeIgniter\Test\CIUnitTestCase;
use CodeIgniter\Test\Mock\MockLogger as LoggerConfig;

final class ErrorlogHandlerTest extends CIUnitTestCase
```

File: tests/system/Log/Handlers/ErrorlogHandlerTest.php [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?php
+namespace CodeIgniter\Log\Handlers;
```
Comment (paulbalandan) [inline tests/system/Log/Handlers/ErrorlogHandlerTest.php:2]:
```suggestion
<?php

namespace CodeIgniter\Log\Handlers;
```

File: tests/system/Log/Handlers/ErrorlogHandlerTest.php [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+	public function testHandle()
+	{
+		$config = new LoggerConfig();
+
+		$config->handlers['CodeIgniter\Log\Handlers\TestHandler']['handles'] = ['critical'];
+
+		$logger = new ErrorlogHandler($config->handlers['CodeIgniter\Log\Handlers\TestHandler']);
+		$this->assertTrue($logger->handle('warning', 'This is a test log'));
+	}
+
```
Comment (paulbalandan) [inline tests/system/Log/Handlers/ErrorlogHandlerTest.php:23]:
```suggestion
```
Remove unnecessary whitespace

**Review by paulbalandan — CHANGES_REQUESTED** [review]

**Comment by lonnieezell** [comment]
@rwillert Are you able to make these revisions?

**Review by paulbalandan — APPROVED** [review]
Actually I have sent a replacement PR due to the inactivity, but since you've worked on this I'll just modify my PR as an enhancement. 😁

**Comment by MGatner** [comment]
Whoops, I didn't notice that! Glad you were still able to use the content.

**Pull request merged**